extensions/community/VoiceRecognition.json

```diff
@@ -135,13 +135,13 @@
       "objectGroups": []
     },
     {
-      "description": "Change voice recognition language.",
```

Comment:
You seem to have removed this action. Is that done on purpose?

Reply:
I'm not sure, "Listen for speeches" existed before.
(look below, it got moved in the file)

There is no action "change language" that exist anymore, and yet the description of the extension says "actions: change language"

Reply:
I removed the change language action, since there is a new optional parameter language on listen speech action
![image](https://user-images.githubusercontent.com/1837825/180378033-c3fcfe47-766e-4137-b598-b16a4faec5d7.png)